coreAPI/src/main/java/net/java/games/input/Event.java

```diff
@@ -38,39 +38,113 @@
  *****************************************************************************/
 package net.java.games.input;
 
+import java.util.Objects;
+
 public final class Event {
+	/** {@link Component} that generated this {@code Event}. */
     private Component component;
+
+	/**
+	 * <p>A value associated with this {@code Event}.</p>
+	 *
+	 * <p>
+	 *     This could represent any number of things, depending on the {@link Component} that generated this
+	 *     {@code Event}. Rotation of a joystick, pressure on a button, etc.
+	 * </p>
+	 */
```

Comment:
This could probably be improved with better examples, if you have any suggestions